### PR TITLE
[10.0][IMP] sale_product_set add image

### PR DESCRIPTION
**Pull request by TDu**
Allow to add an image to a product set and move the product set lines
to a notebook as to simplify adding custom fields to the view.

### CODE REVIEW

**Review by LoisRForgeFlow — APPROVED** [review]
Looks like a good improvement to me :+1:

File: sale_product_set/views/product_set.xml [hunk 1]
```diff
@@ -21,15 +21,24 @@
         <form string="Product set" version="7.0">
```
Comment (JayVora-SerpentCS) [inline sale_product_set/views/product_set.xml]: As this is now for v10, we may remove the version param

**Review by JayVora-SerpentCS — CHANGES_REQUESTED** [review]

**Comment by JayVora-SerpentCS** [comment]
@TDu

**Comment by TDu** [comment]
@JayVora-SerpentCS I removed that version parameter.

**Review by JayVora-SerpentCS — APPROVED** [review]

**Review by bodedra — APPROVED** [review]
LGTM :+1:

File: sale_product_set/models/product_set.py [hunk 2]
```diff
@@ -19,3 +19,7 @@ class ProductSet(models.Model):
         'product_set_id',
         string="Products"
     )
+    image = fields.Binary(
+        string='Image',
+        help='Select an image to display for the product set'
```
Comment (pedrobaeza) [inline sale_product_set/models/product_set.py:25]: Please add `attachment=True` for having the advantages it brings

Comment (TDu) [reply sale_product_set/models/product_set.py:25]: @pedrobaeza Thanks for the hint, I applied the change.

**Comment by pedrobaeza** [comment]
Please forward-port it to v11 (and if you want to migrate it to v12!)

**Pull request merged**